apps/web/pages/event-types/[type].tsx

```diff
@@ -495,7 +496,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
       .string()
       .refine((val) => isValidPhoneNumber(val))
       .optional(),
-    locationLink: z.string().url().optional(), // URL validates as new URL() - which requires HTTPS:// In the input field
+    locationLink: z.string().optional(), // URL validates as new URL() - which requires HTTPS:// In the input field
```

Comment:
Are we not planning to re-add this @sean-brydon ?
```suggestion
    locationLink: z.string().url().optional(), // URL validates as new URL() - which requires HTTPS:// In the input field
```

Reply:
Yes :) I thought I did already - just tried it locally and works fine on all links